cmake/Modules/CompilerConfig.cmake

```diff
@@ -75,6 +76,10 @@ if(OS_WINDOWS AND MSVC)
     /std:c17)
 
   add_link_options(
+    # ToDo figure out how to properly escape this (currently it gets turned into
+    # %%%%PDB%%%%, thanks cmake!)
+    "LINKER:/PDBALTPATH:%_PDB%"
```

Comment:
As this is removed in a later commit, either squash as needed, or remove from here.